core/templates/services/debouncer.service.spec.ts

```diff
@@ -31,7 +31,8 @@ describe('Debouncer service', () => {
     loggerServiceSpy = spyOn(ls, 'log').and.callThrough();
   });
 
-  it('should call a debounced function after a non-zero given wait time',
+  it(
+    'should call a debounced function after a non-zero given wait time',
```

Comment:
Why these changes?

Reply:
These changes were made by mistake when I was testing this file only using fit. When using fit it was showing an error to start from a new line so I did that.

Reply:
Done